packages/shared/src/components/Attachments/AttachmentItem.tsx

```diff
@@ -107,18 +122,19 @@ export const AttachmentItem = forwardRef<HTMLDivElement, AttachmentItemProps>(
                 {isEditing && (
                     <div
                         data-test-id="attachments-actionbar"
-                        className="tw-flex tw-gap-[2px] tw-invisible group-hover:tw-visible"
+                        className="tw-flex tw-gap-[2px] tw-invisible group-focus:tw-visible group-focus-within:tw-visible group-hover:tw-visible"
```

Comment:
```suggestion
                        className="tw-flex tw-gap-0.5 tw-invisible group-focus:tw-visible group-focus-within:tw-visible group-hover:tw-visible"
```
just minor tweak for tailwind classes